styles/styles.scss

```diff
@@ -3,32 +3,69 @@
   --navbar-link-color: rgba(51, 51, 51, 0.7);
   --navbar-link-hover-color: #333333;
   --navbar-link-active-color: #333333;
+  --paragraph-collor: #5f5e5e;
+  --main-blue-color: rgb(54, 120, 244);
   --spacer-smaller: 0.5rem;
   --spacer-small: 0.75rem;
   --spacer-default: 1rem;
   --spacer-big: 1.25rem;
   --spacer-bigger: 1.5rem;
   --spacer-biggest: 2.5rem;
   --font-size-normal: 1.125rem;
+  --font-weight-thicker: 500;
   --font-weight-bold: 700;
 }
 
 body {
   font-family: 'Mulish', sans-serif;
   background-color: var(--navbar-bg-color);
+  margin: var(--spacer-big);
 }
 
+.img-content {
+  text-align: center;
+}
 
-.button {
+.img-content img {
+  max-width: 90%;
+}
 
+.text-content {
```

Comment:
ta klasa będzie całe do poprawy, ale zrobimy to w osobnym tasku, żeby nie blokować